rc/base/ctags.kak

```diff
@@ -7,29 +7,54 @@ decl str-list ctagsfiles 'tags'
 
 def -params 0..1 \
     -shell-candidates '
-        ( for tags in $(printf %s\\n "${kak_opt_ctagsfiles}" | tr \':\' \'\n\'); do
-              namecache=$(dirname ${tags})/.kak.$(basename ${tags}).namecache
-              if [ -z "$(find ${namecache} -prune -newer ${tags})" ]; then
-                  cat ${tags} | cut -f 1 | uniq > ${namecache}
-              fi
-              cat ${namecache}
-          done )' \
+        {
+        oldIFS="$IFS"
+        IFS=:
```

Comment:
You've got a problem when your `ctagsfiles` has white spaces in its path (ex: `/path/to/folder with spaces/tags`). It's ugly but hey, we are dealing with `sh` here! :). Plus we restore IFS so it's fine. I don't think we have many alternatives.